website/docs/r/pi_instance.html.markdown

```diff
@@ -89,6 +89,7 @@ Review the argument references that you can specify for your resource.
 - `pi_replication_scheme` - (Optional, String) The replication scheme that you want to set, either `prefix` or `suffix`.
 - `pi_sap_profile_id` - (Optional, String) SAP Profile ID for the amount of cores and memory.
   - Required only when creating SAP instances.
+- `pi_sap_deployment_type` - (Optional, String) Custom SAP deployment type information (For Internal Use Only).
```

Comment:
if its only for internal use shoudl we document this?

Reply:
We are not disclosing the value this parameter takes so think it is fine.